README.md

```diff
@@ -1212,6 +1220,16 @@ Default: `null`
 
 Type: `string`
 
+##### mssql_ad_sql_user_distname
+
+AD distinguished name to create the [`mssql_ad_sql_user_name`](#mssql_ad_sql_user_name) at.
+
+For example, `CN=sqluser,CN=Users,DC=DOMAIN,DC=COM`.
+
+Default: `null`
```

Comment:
That's a good idea since most users probably still have users in CN=Users. The default then should be as such:
```
CN={{ mssql_ad_sql_user_name }},
CN=Users,
DC={{ ad_integration_realm.split('.') | first | upper }},
{{ 'DC = ' ~ ad_integration_realm.split('.') | second | upper ~ ',' if ad_integration_realm.split('.') | length == 3 else '' }}
DC={{ ad_integration_realm.split('.') | third | upper }}
```

Reply:
@JasonN3 can realm FQDN consist of more than three portions, i.e. can it be in the form `cortoso.example.domain.com`?

Reply:
> @JasonN3 can realm FQDN consist of more than three portions, i.e. can it be in the form `cortoso.example.domain.com`?

I'm pretty sure from my past experience with LDAP and AD that it can have several components

Reply:
That is - try to find a solution that given any `a.b.c.d.e.f.g......y.z` can convert that to `DC=a,DC=b,....,DC=y,DC=z`

Reply:
```
ansible -c local -i localhost, all -e hn=a.b.c.d.e.f.g -m debug -a 'msg={{hn.split(".")|map("regex_replace","^","DC=")|join(",")}}'
localhost | SUCCESS => {
    "msg": "DC=a,DC=b,DC=c,DC=d,DC=e,DC=f,DC=g"
}ansible -c local -i localhost, all -e hn=a.b.c.d.e.f.g -m debug -a 'msg={{hn.split(".")|map("regex_replace","^","DC=")|join(",")}}'
localhost | SUCCESS => {
    "msg": "DC=a,DC=b,DC=c,DC=d,DC=e,DC=f,DC=g"
}
```

Reply:
The domain can be of any length so using the map is definitely the better approach. Having it default to the Users OU is fine as long as there is an option to put it elsewhere.

Reply:
I applied Rich's suggestion, thanks